test/kafo/answer_file_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+
+describe 'Kafo::AnswerFile' do
```

Comment:
Generally you actually use the class reference.
```suggestion
describe Kafo::AnswerFile do
```